web_auto_refresh/readme.rst

```diff
@@ -0,0 +1,56 @@
+web auto refresh
```

Comment:
You have to copy raw version instead: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst